client/src/modules/vouchers/toolkit/patient_fled/patient_fled.js

```diff
@@ -0,0 +1,193 @@
+angular.module('bhima.controllers')
+  .controller('PateintFledKitController', PateintFledKitController);
+
+PateintFledKitController.$inject = [
+  '$uibModalInstance', 'DebtorGroupService', 'NotifyService',
+  'SessionService', 'bhConstants', '$translate', 'VoucherToolkitService',
+];
+
+// Import transaction rows for a convention payment
+function PateintFledKitController(
+  Instance, DebtorGroup, Notify, Session, bhConstants, $translate, ToolKits,
+) {
+  const vm = this;
+
+  const { MAX_DECIMAL_PRECISION } = bhConstants.precision;
+
+  // global variables
+  vm.debtorGroupFilter = { is_convention : 1 };
+  vm.gridOptions = {};
+  vm.account = {};
+  vm.enterprise = Session.enterprise;
+
+  // expose to the view
+  vm.selectGroupInvoices = selectGroupInvoices;
+  vm.close = Instance.close;
+  vm.import = submit;
+
+  // Set up page elements data (debtor select data)
+  vm.onSelectDebtor = onSelectDebtor;
+  vm.onSelectAccount = onSelectAccount;
+
+  function onSelectDebtor(debtorGroup) {
```

Comment:
I made a mistake in the issue - this shouldn't select a debtor group, but a debtor instead.  We will want to use `bhFindPatient` instead of convention, then show that patient's invoices.  I had forgotten that convention payment doesn't work this way, and instead has you select an entire debtor group.